src/SurveyService.Validation/UserAnswer/CreateUserAnswerRequestValidator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.UserAnswer;
+using LT.DigitalOffice.SurveyService.Validation.UserAnswer.Interfaces;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LT.DigitalOffice.SurveyService.Validation.UserAnswer;
+
+public class CreateUserAnswerRequestValidator : AbstractValidator<CreateUserAnswerRequest>, ICreateUserAnswerRequestValidator
+{
+  public CreateUserAnswerRequestValidator(
+    IOptionRepository _optionRepository,
+    IHttpContextAccessor _httpContextAccessor)
+  {
+    RuleFor(request => new
+    {
+      optionsIds = request.OptionIds,
+      options = _optionRepository.GetByIdsAsync(request.OptionIds).Result
+    })
+      .Cascade(CascadeMode.Stop)
+      .Must(x => x.optionsIds.Any())
```

Comment:
I think, don't need it here; [Required] should catch empty list
please, check it

Reply:
Required will catch empty line, but not an empty list. Check https://docs.microsoft.com/ru-ru/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=net-6.0